examples/diffusion/textual_inversion_stable_diffusion/detsd/utils.py

```diff
@@ -0,0 +1,59 @@
+from typing import List, Sequence, Tuple
+
+import torch
+import torch.nn as nn
+
+
+def add_new_tokens_to_tokenizer(
+    concept_token: str,
+    initializer_tokens: Sequence[str],
+    tokenizer: nn.Module,
+) -> Tuple[List[int], List[int], str]:
+    """Helper function for adding new tokens to the tokenizer and extending the corresponding
+    embeddings appropriately, given a single concept token and its sequence of corresponding
+    initializer tokens.  Returns the lists of ids for the initializer tokens and their dummy
+    replacements, as well as the string representation of the dummies.
+    """
+    initializer_ids = tokenizer(
+        initializer_tokens,
+        padding="max_length",
+        truncation=True,
+        max_length=tokenizer.model_max_length,
+        return_tensors="pt",
+        add_special_tokens=False,
+    ).input_ids
```

Comment:
Agree, this was the other piece of notation imported from the [HF example](https://huggingface.co/docs/diffusers/v0.4.1/en/training/text_inversion). Will just change it, rather than stick with the original nomenclature.